src/web/routes/application/check/get.js

```diff
@@ -32,7 +33,7 @@ const getCheck = (steps) => (req, res) => {
   req.session.nextAllowedStep = stateMachine.dispatch(actions.GET_NEXT_PATH, req, steps)
 
   res.render('check', {
-    claim: req.session.claim,
+    children: req.session.children,
```

Comment:
Where does the rest of the claim come from for the other fields on the check page if this has changed to `children` rather than `claim`?

Reply:
Correct - `claim` was never used. All other data comes from `checkRowData`